addons/medical_status/functions/fnc_isInStableCondition.sqf

```diff
@@ -17,7 +17,22 @@
 
 params ["_unit"];
 
-alive _unit
-&& {!IS_UNCONSCIOUS(_unit)}
-&& {GET_WOUND_BLEEDING(_unit) == 0}
-&& {_unit call FUNC(hasStableVitals)}
+if (!alive _unit) exitWith { false };
+if (IS_UNCONSCIOUS(_unit)) exitWith { false };
+if (IS_BLEEDING(_unit)) exitWith { false };
+if (GET_HEART_RATE(_unit) < EGVAR(medical_treatment,minHeartRatePAK)) exitWith { false };
```

Comment:
```suggestion
if (!alive _unit) exitWith {false};
if (IS_UNCONSCIOUS(_unit)) exitWith {false};
if (IS_BLEEDING(_unit)) exitWith {false};
if (GET_HEART_RATE(_unit) < EGVAR(medical_treatment,minHeartRatePAK)) exitWith {false};
```

I'm not sure if all these if's will not make the function much slower. Having it as `<cond> && {<cond>} & ...` with that many conditions will look ugly tho. 🤔

Reply:
```
if (!alive _unit
    || {IS_UNCONSCIOUS(_unit)}
    || {IS_BLEEDING(_unit)}
    || {GET_HEART_RATE(_unit) < EGVAR(medical_treatment,minHeartRatePAK)}
) exitWith {false};
```

It's not that bad